pytest/tests/sanity/lightclnt.py

```diff
@@ -56,7 +59,7 @@ def get_light_client_block(hash_, last_known_block):
     global block_producers_map
 
     ret = nodes[0].json_rpc('next_light_client_block', [hash_])
-    if ret['result'] is not None and last_known_block is not None:
+    if ret['result'] != {} and last_known_block is not None:
```

Comment:
```suggestion
    if ret['result'] and last_known_block is not None:
```

This'd be more Pythonic, but not sure if we use falsy idioms in our Python code.

Reply:
We generally don't have as defined guidelines for python code. I personally prefer explicit comparisons.